data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -37,13 +37,38 @@ WITH t_hypothesis_annotation_with_doi AS (
   AND created >= '2022-09-01' -- to ignore any public reviews posted before Sep 2022
 ),
 
+t_manual_osf_preprint_match AS(
+  SELECT
+    * EXCEPT(rn)
+  FROM
+  (
+    SELECT 
+      *,
+      ROW_NUMBER() OVER(PARTITION BY manuscript_id ORDER BY imported_timestamp DESC) AS rn
+    FROM `elife-data-pipeline.prod.unmatched_manuscripts`
+    WHERE osf_preprint_url IS NOT NULL
+  )
+  WHERE rn=1
+),
+
+t_hypothesis_annotation_with_osf_doi AS(
```

Comment:
We should avoid pulling in hypothesis annotation from a revision into the first version. To avoid that we may want to add a date filter.

See also https://github.com/elifesciences/data-hub-issues/issues/725